<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCRDPrintWhenPanelUI</class>
 <widget class="QWidget" name="NCRDPrintWhenPanelUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="1">
    <widget class="QGroupBox" name="gbVisOpt">
     <property name="title">
      <string>Advanced Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="2" column="3">
       <widget class="QLineEdit" name="functionID"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelItemId">
        <property name="text">
         <string>Item ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="labelZoneId">
        <property name="text">
         <string>Zone ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="zoneID"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="itemID"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelFunctionId">
        <property name="text">
         <string>Function ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Print When:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>HTML Options:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="printWhen">
        <property name="toolTip">
         <string>Print only when expression is true</string>
        </property>
        <property name="whatsThis">
         <string>Print only when expression is true</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLineEdit" name="htmlOptions"/>
      </item>
      <item row="1" column="6">
       <widget class="QToolButton" name="btnEditHtmlOpt">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="btnEditPrintWhen">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>printWhen</tabstop>
  <tabstop>btnEditPrintWhen</tabstop>
  <tabstop>htmlOptions</tabstop>
  <tabstop>btnEditHtmlOpt</tabstop>
  <tabstop>itemID</tabstop>
  <tabstop>functionID</tabstop>
  <tabstop>zoneID</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
